_docs_use-cases/use-case-consumption-delete-a-third-party-owned-relationshipattribute-and-notify-peer.md

```diff
@@ -36,3 +36,26 @@ require:
 required_by:
 # End automatic generation
 ---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a third party owned RelationshipAttribute, i.e. a [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) that exists in a Relationship with a peer, but is not owned by you or this peer but by a third party.
```

Comment:
In the "delete peer shared Attribute" use case, I'd think we should also emphasize who the owner of the Attribute should be in the first sentence. This Attribute must then be owned by the peer, but not by a third party, right?